pages/linux/limactl.md

```diff
@@ -0,0 +1,29 @@
+# limactl
+
+> Virtual machine manager for linux guests, with multiple vm templates available.
+> Can be used to run containers on macos, but also for generic virtual machine use-cases on macos and linux hosts.
+> More information: <https://github.com/lima-vm/lima>.
+
+- Pick any of the available distributions, see templates in `/usr/local/share/lima/templates/`:
+
+`LIMA_DISTRO=debian|fedora|ubuntu|…`
```

Comment:
```suggestion
`LIMA_DISTRO={{debian|fedora|ubuntu|…}}`
```
We use `{{}}` in our pages so that our clients can highlight them as placeholders.